ArcadiaParties-SPA/src/app/content/content.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
We decided with @Duveshka that you don't have to spend time for tests in this specific project.

Reply:
spec files removed